README.md

```diff
@@ -96,15 +96,49 @@ pip install -e ".[test,dev]"
 pip install -e plugins/fuzzyhashes
 ```
 
+## Settings
+
+Surfactant settings can be changed using the `surfactant config` subcommand, or by hand editing the settings configuration file (this is not the same as the JSON file used to configure settings for a particular sample that is described later).
+
+### Command Line
+
+Using `surfactant config` is very similar to basic use of `git config`. The key whose value is being accessed will be in the form `section.option` where `section` is typically a plugin name or `core`, and `option` is the option to set. As an example, a plugin that includes an entire copy of the original files as part of the generated SBOM might provide an option to specify the format.
+
+Setting the configuration option to `base64` could be done using:
+
+```bash
+surfactant config include_entire_file.format base64
+```
+
+Getting the currently set value for the option would then be done with:
+
+```bash
+surfactant config include_entire_file.format
+```
+
+### Manual Editing
+
+If desired, the settings config file can also be manually edited. The location of the file will depend on your platform.
+On Unix-like platforms (including macOS), the XDG directory specification is followed and settings will be stored in
+`${XDG_CONFIG_HOME}/surfactant/config.toml`. If the `XDG_CONFIG_HOME` environment variable is not set, the location defaults
+to `~/.config`. On Windows, the file is stored in the Roaming AppData folder at `%APPDATA%\\surfactant\\config.toml`.
+
+The file itself is a TOML file, and for the previously mentioned example plugin may look something like this:
+
+```toml
+[include_entire_file]
+format = "base64"
+```
+
 ## Usage
 
 ### Identify sample file
 
 In order to test out surfactant, you will need a sample file/folder. If you don't have one on hand, you can download and use the portable .zip file from <https://github.com/ShareX/ShareX/releases> or the Linux .tar.gz file from <https://github.com/GMLC-TDC/HELICS/releases>. Alternatively, you can pick a sample from https://lc.llnl.gov/gitlab/cir-software-assurance/unpacker-to-sbom-test-files
 
-### Build configuration file
+### Build sample configuration file
 
-A configuration file contains the information about the sample to gather information from. Example JSON configuration files can be found in the examples folder of this repository.
+A sample configuration file contains the information about the sample to gather information from. Example JSON sample configuration files can be found in the examples folder of this repository.
```

Comment:
```suggestion
A configuration file for a sample contains the information about the sample to gather information from. Example JSON sample configuration files can be found in the examples folder of this repository.
```

Reply:
It becomes more clear when you read the rest of the documentation, so this change might not be neccessary